pkgs/by-name/te/technitium-dns-server/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  dotnet-sdk_8,
+}:
+stdenv.mkDerivation rec {
```

This doesn't need a C Compiler
```suggestion
stdenvNoCC.mkDerivation rec {
```

---

Great, thanks! I hadn't heard about that before, it's nice to learn new things! Fixed this.